docs/utility-methods.md

```diff
@@ -4,6 +4,53 @@ sidebar_position: 5
 
 # Utility Methods
 
+## `makeSafeQueryRunner`
+
+A wrapper around `q` so you can easily use `groqd` with an actual fetch implementation, with a sprinkle of sugar to improve error message legibility.
+
+Pass `makeSafeQueryRunner` a "query executor" of the shape `type QueryExecutor = (query: string, ...rest: any[]) => Promise<any>`, and it will return a "query runner" function. This is best illustrated with an example:
+
+```ts
+import sanityClient from "@sanity/client";
+import { q } from "groqd";
+
+// Wrap sanityClient.fetch
+const client = sanityClient({
+  /* ... */
+});
+export const runQuery = makeSafeQueryRunner((query) => client.fetch(query));
+
+// 👇 Now you can run queries and `data` is strongly-typed, and runtime-validated.
+const data = await runQuery(
+  q("*").filter("_type == 'pokemon'").grab({ name: q.string() }).slice(0, 150)
+);
+```
+
+In Sanity workflows, you might also want to pass e.g. params to your `client.fetch` call. To support this, add additional arguments to your `makeSafeQueryRunner` argument's arguments as below.
```

Comment:
This sentence is a bit strange, aren't you always using Groqd in a Sanity workflow? Also I think we can drop the , after To support this since it's one complete thought. Finally argument's arguments is a bit dizzying.

Reply:
Good callout. This is a pretty old sentence, i'll just rework it.